app/controllers/serviceProviderButtonBar.js

```diff
@@ -0,0 +1,55 @@
+var args = arguments[0] || {};
+var countySelector = require("countySelectorUtils");
+
+var providerListObject;
+
+exports.setProviderListObject = function(listObject){
+	providerListObject = listObject;
+};
+
+/**
+ * Function that either displays list of service providers or map of
+ * service providers based on user preference. 
+ */
+function toggleMapListView(){
+	var mapModule = providerListObject.getMapViews().mapModule;
+	var map = providerListObject.getMapViews().map;
+	var listView = providerListObject.getListView();
+	
+	if(mapModule.visible){
+		setMapVisibility(false);
+	}
+	else{
+		setMapVisibility(true);
+		map.setRegion({latitude:39.719704, longitude:-84.219832, latitudeDelta:0.2, longitudeDelta:0.2});
```

Comment:
Yeah - it's definitely something we need to flesh out a bit more, so I'll add a TODO. The service providers that we are using currently are all in the Dayton area, hence the hard-coded lat/long. But since users of the app might have geolocation disabled (to try to keep their batterer from tracking them), using their location might not be the best solution. Using a combination of service provider locations and person location (if available) would probably be the best solution, but I'd have to think that one thru a bit more.